### PR TITLE
Newly added files

### PR DESCRIPTION
**Pull request by snkzt**
WHAT about this PR: New files added to app
WHY is files should be added: they are necessary to upload the app on web
Added files: Procfile, index.js, package.json and .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Node build artifacts
+node_modules/
+npm-debug.log
+
+# Local development
+*.env
+*.dev
+.DS_Store
+
+# Docker
```
Comment (VeysonD) [inline .gitignore]: Does your application use Docker?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Dockerfile
+docker-compose.yml
```
Comment (VeysonD) [inline .gitignore]: Same as above

Comment (VeysonD) [inline .gitignore]: It's best to add a new line at the end of each file

File: Procfile [hunk 3]
```diff
@@ -0,0 +1 @@
+web: npm start
```
Comment (VeysonD) [inline Procfile]: It's best to add a new line at the end of each file

File: index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const express = require('express')
+const path = require('path')
+const PORT = process.env.PORT || 5000
+const app = express()
+
+app.use(express.static(path.join(__dirname, 'assets')))
+app.get('/', (req, res) => res.sendFile(path.join(__dirname,'assets/html/home.html')))
+app.listen(PORT, () => console.log(`Listening on ${ PORT }`))
```
Comment (VeysonD) [inline index.js]: It's best to add a new line at the end of each file

File: package.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "user-information-database",
+  "version": "1.0.0",
+  "description": "*MAKE YOUR WISH TO MAKE IT HAPPEN*",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (VeysonD) [inline package.json:7]: Your procfile says `npm start` but I don't see a `start` script defined here

File: .gitignore (hunk 1 above)
Comment (snkzt) [reply .gitignore]: No but not sure to use it so still leaving it.
But I will delete this.

File: .gitignore (hunk 2 above)
Comment (snkzt) [reply .gitignore]: got it

File: package.json (hunk 5 above)
Comment (snkzt) [reply package.json:7]: I changed the Procfile to node index.js

**Pull request merged**